processor/src/config/payment-method.config.ts

```diff
@@ -13,7 +14,10 @@ export type PaymentMethodConfig = {
 };
 
 export const paymentMethodConfig: PaymentMethodConfig = {
-  bancontactcard: {
```

Comment:
This config is used in the notification converter, which works based on the Adyen keys instead of our keys of the payment-methods. And so this needed correcting for the existing bancontactcard